### ISSUE TITLE
Remove scope

### ISSUE THREAD

**Issue by Flohack74** [ubports/telegram-app#71]
As we are heading towards removing scopes from the OS also the Telegram scope needs to go.

**Comment by ChloeWolfieGirl** [comment]
The latest q&a Marius said we might keep the scopes, and if we dont keep the scopes it mightbbe best to keep the code until we know what theyre being replaced with, some of the code could still be used for the replacement.

Personally I hope to see telegram contacts in the today scope or on my homescreen in someway.

**Comment by Flohack74** [comment]
Yes, its not on the issues someone is working on. And its targeted for 3.0 which is a huge rewrite, so no worries ;)
